.travis.yml

```diff
@@ -36,6 +36,7 @@ after_script:
     - docker logs blogtest
     - docker stop blogtest
     - docker rm blogtest
+    - exit 0 
```

Comment:
Strange to see that we must do this. In the Travis documentation there is

> The exit code of `after_success`, `after_failure`, `after_script` and subsequent stages do not affect the build result. However, if one of these stages times out, the build is marked as a failure.

So I think we have to check why the aster script is not working correctly without putting an Exit 0.
I'm going to check the failure build of your PullRequest